src/home_robot_hw/home_robot_hw/nodes/contact_graspnet_server.py

```diff
@@ -25,12 +25,13 @@
 sys.path.append(BASE_DIR)
 import config_utils  # noqa: E402
 from contact_grasp_estimator import GraspEstimator  # noqa: E402
+from visualization_utils import show_image, visualize_grasps  # noqa: E402
+
 from data import (  # noqa: E402
     depth2pc,
     load_available_input_data,
     regularize_pc_point_count,
 )
-from visualization_utils import show_image, visualize_grasps  # noqa: E402
```

Comment:
this looks like a linter issue - probably wrong version of libraries in pre-commit

Reply:
hmm, I'm also using 3.3.2 which is the same as what being installed in circleCI...

Reply:
Yes, this is a known issue - this causes CircleCI tests to fail. I remove these changes from my commit, whenever this happens. @zephirefaith did you find a better fix for this?

Reply:
Removed them for now